python/cugraph/structure/graph.py

```diff
@@ -341,97 +100,109 @@ def from_cudf_edgelist(
+            This cudf.Series wraps a gdf_column of size E (E: number of edges).
+            The gdf column contains the destination index for each edge.
+            Destination indices must be in the range [0, V) (V: number of
+            vertices).
+        value_col : cudf.Series, optional
+            This pointer can be ``None``.
+            If not, this cudf.Series wraps a gdf_column of size E (E: number of
+            edges).
+            The gdf column contains the weight value for each edge.
+            The expected type of the gdf_column element is floating point
+            number.
+        Examples
+        --------
+        >>> gdf = cudf.read_csv('datasets/karate.csv', delimiter=' ',
+        >>>                   dtype=['int32', 'int32', 'float32'], header=None)
+        >>> M = gdf.to_pandas()
+        >>> M = scipy.sparse.coo_matrix((M['2'],(M['0'],M['1'])))
+        >>> M = M.tocsr()
+        >>> offsets = cudf.Series(M.indptr)
+        >>> indices = cudf.Series(M.indices)
+        >>> G = cugraph.Graph()
+        >>> G.from_cudf_adjlist(offsets, indices, None)
+        """
+        if self.Impl is None:
+            self.Impl = simpleGraphImpl(self.graph_properties)
+        self.Impl._simpleGraphImpl__from_adjlist(offset_col,
+                                                 index_col,
+                                                 value_col)
 
-        self.renumber_map = renumber_map
+    def from_dask_cudf_edgelist(
+        self,
+        input_ddf,
+        source="source",
+        destination="destination",
+        edge_attr=None,
+        renumber=True,
+    ):
+        """
+        Initializes the distributed graph from the dask_cudf.DataFrame
+        edgelist. Undirected Graphs are not currently supported.
+        By default, renumbering is enabled to map the source and destination
+        vertices into an index in the range [0, V) where V is the number
+        of vertices.  If the input vertices are a single column of integers
+        in the range [0, V), renumbering can be disabled and the original
+        external vertex ids will be used.
+        Note that the graph object will store a reference to the
+        dask_cudf.DataFrame provided.
+        Parameters
+        ----------
+        input_ddf : dask_cudf.DataFrame
+            The edgelist as a dask_cudf.DataFrame
+        source : str or array-like
+            source column name or array of column names
+        destination : str
+            destination column name or array of column names
+        edge_attr : str
+            weights column name.
+        renumber : bool
+            If source and destination indices are not in range 0 to V where V
+            is number of vertices, renumber argument should be True.
+        """
+        if self.Impl is None:
+            self.Impl = simpleDistributedGraphImpl(self.graph_properties)
+        self.Impl._simpleDistributedGraphImpl__from_edgelist(input_ddf,
```

Comment:
This is done to make from_edgelist private. Since it is internally called and not intended to be exposed to user, forgoing polymorphism is an acceptable trade off. It can be changed in future updates if needed